js/ui/text_box/ui.text_editor.base.js

```diff
@@ -32,6 +33,7 @@ const ALLOWED_STYLE_CLASSES = [
     TEXTEDITOR_STYLING_MODE_PREFIX + 'filled',
     TEXTEDITOR_STYLING_MODE_PREFIX + 'underlined'
 ];
+const TEXTEDITOR_COMPRESSED_CLASS = 'dx-texteditor-compressed';
```

Comment:
What does the name of this class mean?

Reply:
It means we should use flex-basis: 0; style for texteditor-input-container to minimize the input container width

Reply:
compress -> collapse?